demo/package.json

```diff
@@ -10,7 +10,7 @@
     "ios": "npx expo start --ios",
     "eject": "npx expo eject",
     "test": "npx jest",
-    "lint": "npx prettier --write '{*.js,**/*.js,*.jsx,**/*.jsx}' && npx eslint '{*.js,**/*.js,*.jsx,**/*.jsx}'"
+    "lint": "npx prettier --write '{*.js,**/*.js,*.jsx,**/*.jsx}' && npx eslint '{*.js,**/*.js,*.jsx,**/*.jsx,**/*.snap}'"
```

Comment:
Can you expand on the choice of linting the snapshots? I think it seems reasonable, but I haven't given this much thought before.

Reply:
I'm not sure if it is a good idea but I saw an eslint rule to keep snapshots small. https://github.com/tasitlabs/tasit/issues/14#issuecomment-448313550
What do you think?

Reply:
Ahhhh got it. I definitely wanted to apply the rule to keep snapshots small - it just didn't cross my mind when reading this line that this change is what would make that possible. Sounds good!